packages/playwright-ct-core/index.d.ts

```diff
@@ -56,4 +56,4 @@ export function defineConfig(config: PlaywrightTestConfig, ...configs: Playwrigh
 export function defineConfig<T>(config: PlaywrightTestConfig<T>, ...configs: PlaywrightTestConfig[]): PlaywrightTestConfig<T>;
 export function defineConfig<T, W>(config: PlaywrightTestConfig<T, W>, ...configs: PlaywrightTestConfig[]): PlaywrightTestConfig<T, W>;
 
-export { expect, devices } from 'playwright/test';
+export * from 'playwright/test';
```

Comment:
- You should export this on the first line, so that any overrides take precedence, for example custom `TestType` and `defineConfig`.
- Should there be a non-`.d.ts` change as well? I don't think `pnpm` is worried about types.